packages/toolboxes/substrate/src/toolbox/baseSubstrateToolbox.ts

```diff
@@ -15,7 +15,11 @@ import { Network, type SubstrateNetwork } from "../types/network.ts";
 type SubstrateTransferParams = {
   recipient: string;
   assetValue: AssetValue;
-  from?: string;
+  from: string;
+};
+
+export const isKeyringPair = (account: IKeyringPair | Signer): account is IKeyringPair => {
+  return (account as IKeyringPair).sign !== undefined;
 };
```

Comment:
If `account is IKeyringPair` I think `as ` statement is obsolete 🤔 
```suggestion
export const isKeyringPair = (account: IKeyringPair | Signer): account is IKeyringPair => {
  return !account.sign
};
```

Reply:
Fixed